src/components/UserProfile.js

```diff
@@ -12,28 +12,32 @@ class UserProfile extends React.Component {
     const { user, cart } = this.props;
     return (
       <div id='userProfile'>
-        <h3>
-          User profile for {user.firstName} {user.lastName}
-        </h3>
-        <div id='userInfo'>
-          First Name: {user.firstName}<br/>
-          Last Name: {user.lastName}<br/>
-          Class: {user.class}<br/>
-          Email:{user.email}<br/>
+        <div id='pad-left'>
+          <h3>
+            User profile for {user.firstName}  {user.lastName}
+          </h3>
+          <div id='userInfo'>
+            First Name: {user.firstName}<br/>
+            Last Name: {user.lastName}<br/>
+            Class: {user.class}<br/>
+            Email:{user.email}<br/>
+          </div>
+          <Link to="/edit-user">Edit Profile Info</Link>
+          <div className='admin-tools'>
+            {user.isAdmin ? <a href="/edit-user">Edit Current Users</a> : ''}
+          </div>
```

Comment:
Links not being specified as <Link> will hard reload our application. This should not be a major problem, but is that what we want to do here?

Reply:
Oh..lemme double check again locally

Reply:
Items are persisting in cart while using <Link>
Totally optional to push to Herkou, I'm nervous about breaking things before presentation too haha